ssl/ssl_lib.c

```diff
@@ -1150,7 +1151,9 @@ void SSL_free(SSL *s)
     ssl_free_wbio_buffer(s);
 
     BIO_free_all(s->wbio);
+    s->wbio = NULL;
     BIO_free_all(s->rbio);
+    s->rbio = NULL;
```

Comment:
I wonder why this is necessary since we later free `s` anyway.

Reply:
It's a good habit to set pointers to NULL, just in case we get a double free anything funny. I could remove it if you'd like?